js/number/Index.svelte

```diff
@@ -96,7 +102,7 @@
 		border: var(--input-border-width) solid var(--input-border-color);
 		border-radius: var(--input-radius);
 	}
-	input[type="number"] {
+	input[type="text"] {
```

Comment:
This means that browsers won't treat this field as numeric. On Desktop it means we'll lose these:

<img width="470" alt="image" src="https://github.com/user-attachments/assets/f5d8df72-5322-4606-870f-d24ac566e429">

On mobile, I think it will lead to a ux degradation since the numeric keyboard won't pop up anymore

Reply:
I don't think this change should be needed since at least according to the original issue, it is in theory possible to set a numeric field blank:

> Currently, I can set the value to Callable, where Callable returns None. I have tested this, and the frontend displays it as blank, which aligns with my requirements.